### PR TITLE
Add ArangoSearch Geo Search Examples

### PR DESCRIPTION
**Pull request by Simran-B**
Todo: Apply to 3.9 after review

https://deploy-preview-728--zealous-morse-14392b.netlify.app/docs/3.8/arangosearch-geospatial-search.html

### CODE REVIEW

File: 3.8/arangosearch-geospatial-search.md [hunk 1]
```diff
@@ -1,13 +1,324 @@
...
+
+Create a `geopoint` Analyzer in arangosh to pre-process raw coordinates with
+latitude and longitude stored in two different attributes. These attributes
+cannot be at the top-level of the document, but must be nested in an object,
+e.g. `{ location: { lat: 40.78, lon: -73.97 } }`. The path relative to the
+parent attribute (here: `location`) needs to be described in the Analyzer
+properties for each of the coordinate attributes:
+
+```js
+//db._useDatabase("your_database"); // Analyzer will be created in current database
```
Comment (gnusi) [inline 3.8/arangosearch-geospatial-search.md:58]: leftover?

**Review by gnusi — APPROVED** [review]

File: 3.8/arangosearch-geospatial-search.md (hunk 1 above)
Comment (Simran-B) [reply 3.8/arangosearch-geospatial-search.md:58]: No, this is on purpose to remind users that they may want to change to the appropriate database.

**Comment by joerg84** [comment]
@Simran-B is this ready to be merged?

**Comment by Simran-B** [comment]
I need to apply the changes to all versions, otherwise it's ready. Will get it merged later today.

**Pull request merged**